backend/apps/projects/migrations/0001_initial.py

```diff
@@ -8,29 +12,79 @@ class Migration(migrations.Migration):
+                ('visibility', models.CharField(choices=[('public', 'Public'), ('private', 'Private'), ('unlisted', 'Unlisted')], default='public', max_length=20)),
                 ('created_at', models.DateTimeField(auto_now_add=True)),
                 ('updated_at', models.DateTimeField(auto_now=True)),
+                ('startup', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='projects', to='startups.startupprofile')),
             ],
             options={
                 'verbose_name': 'Project',
                 'verbose_name_plural': 'Projects',
+                'ordering': ['-created_at'],
             },
         ),
+        migrations.CreateModel(
+            name='ProjectAttachment',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('file', models.FileField(upload_to='project_attachments/%Y/%m')),
+                ('type', models.CharField(max_length=20)),
+                ('caption', models.CharField(blank=True, max_length=255)),
+                ('order', models.PositiveIntegerField(default=0)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('project', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='attachments', to='projects.project')),
+            ],
+            options={
+                'ordering': ['order', 'created_at'],
+            },
+        ),
+        migrations.CreateModel(
+            name='ProjectAudit',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('action', models.CharField(max_length=20)),
+                ('timestamp', models.DateTimeField(auto_now_add=True, db_index=True)),
+                ('changes', models.JSONField()),
+                ('user_agent', models.TextField(blank=True)),
+                ('project', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='audit_logs', to='projects.project')),
+                ('user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'ordering': ['-timestamp'],
+            },
+        ),
+        migrations.AddIndex(
+            model_name='project',
+            index=models.Index(fields=['startup', 'status'], name='projects_pr_startup_2b2945_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='project',
+            index=models.Index(fields=['created_at'], name='projects_pr_created_6b02e3_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='projectaudit',
+            index=models.Index(fields=['project', '-timestamp'], name='projects_pr_project_0e7a37_idx'),
+        ),
```

Comment:
Using '-timestamp' as a string in Index(fields=...) is invalid. Django expects field names (strings) or expressions; prefixing with '-' is not supported in the fields list and will raise errors when running migrations.

Use an Index with expressions for descending order or omit descending ordering. For Django >= 3.2 you can use expressions:
```
from django.db.models import F
migrations.AddIndex(
    model_name='projectaudit',
    index=models.Index(fields=['project', 'timestamp'], name='projects_pr_project_ts_idx')
)
```
Or use models.Index(fields=['project', 'timestamp']) and rely on ordering in queries, or use models.Index(models.F('timestamp').desc()) if supported by your Django version. Update both migrations and model Meta.indexes.